2018-10/workshop-two/main.go

```diff
@@ -0,0 +1,36 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"net/http"
+	"os"
+
+	"github.com/edmontongo/darksky"
+)
+
+func main() {
+	var secret string
+	// define and parse out command line flags
+	flag.StringVar(&secret, "secret", "", "DarkSky API key (required)")
+	port := flag.String("port", "3001", "Port the service will be listening on")
```

Comment:
are you intentionally using two different methods of getting flags (String vs. StringVar)?